enclave/sgx/revocationinfo.c

```diff
@@ -166,3 +166,178 @@ void oe_free_get_revocation_info_args(oe_get_revocation_info_args_t* args)
+        {
+            if (!(in.tcb_info = realloc(in.tcb_info, out.tcb_info_size)))
+            {
+                OE_RAISE(OE_OUT_OF_MEMORY);
+            }
+
+            in.tcb_info_size = out.tcb_info_size;
+        }
+
+        /* tcb_issuer_chain */
+        if (in.tcb_info_issuer_chain_size < out.tcb_info_issuer_chain_size)
+        {
+            if (!(in.tcb_info_issuer_chain = realloc(
+                      in.tcb_info_issuer_chain,
+                      out.tcb_info_issuer_chain_size)))
+            {
+                OE_RAISE(OE_OUT_OF_MEMORY);
+            }
+
+            in.tcb_info_issuer_chain_size = out.tcb_info_issuer_chain_size;
+        }
+
+        /* pck crl */
+        if (in.pck_crl_size < out.pck_crl_size)
+        {
+            if (!(in.pck_crl = realloc(in.pck_crl, out.pck_crl_size)))
+            {
+                OE_RAISE(OE_OUT_OF_MEMORY);
+            }
+
+            in.pck_crl_size = out.pck_crl_size;
+        }
+
+        /* root ca crl */
+        if (in.root_ca_crl_size < out.root_ca_crl_size)
+        {
+            if (!(in.root_ca_crl =
+                      realloc(in.root_ca_crl, out.root_ca_crl_size)))
+            {
+                OE_RAISE(OE_OUT_OF_MEMORY);
+            }
+
+            in.root_ca_crl_size = out.root_ca_crl_size;
+        }
+
+        /* pck crl issuer chain */
+        if (in.pck_crl_issuer_chain_size < out.pck_crl_issuer_chain_size)
+        {
+            if (!(in.pck_crl_issuer_chain = realloc(
+                      in.pck_crl_issuer_chain, out.pck_crl_issuer_chain_size)))
+            {
+                OE_RAISE(OE_OUT_OF_MEMORY);
+            }
+
+            in.pck_crl_issuer_chain_size = out.pck_crl_issuer_chain_size;
+        }
+
+        /* qe id */
+        if (in.qe_identity_size < out.qe_identity_size)
+        {
+            if (!(in.qe_identity =
+                      realloc(in.qe_identity, out.qe_identity_size)))
+            {
+                OE_RAISE(OE_OUT_OF_MEMORY);
+            }
+
+            in.qe_identity_size = out.qe_identity_size;
+        }
+
+        /* qe id issuer chain*/
+        if (in.qe_identity_issuer_chain_size <
+            out.qe_identity_issuer_chain_size)
+        {
+            if (!(in.qe_identity_issuer_chain = realloc(
+                      in.qe_identity_issuer_chain,
+                      out.qe_identity_issuer_chain_size)))
+            {
+                OE_RAISE(OE_OUT_OF_MEMORY);
+            }
+
+            in.qe_identity_issuer_chain_size =
+                out.qe_identity_issuer_chain_size;
+        }
+    }
+
+    OE_CHECK((oe_result_t)retval);
+
+    *args = out;
+    memset(&out, 0, sizeof(out));
+    result = OE_OK;
+
+done:
+
+    /* Free buffers. */
+    if (result != OE_OK)
+    {
+        oe_free_quote_verification_collateral_args(&in);
+        oe_free_quote_verification_collateral_args(&out);
+    }
+
+    return result;
+}
+
+void oe_free_quote_verification_collateral_args(
```

Comment:
add sgx into function name for consistency with line 330

Reply:
Thanks, function name changed.